src/nodes/mv_normal_weighted_mean_precision.jl

```diff
@@ -5,15 +5,15 @@ import StatsFuns: log2π
 @average_energy MvNormalWeightedMeanPrecision (q_out::Any, q_ξ::Any, q_Λ::Any) = begin
     m_mean, v_mean = mean_cov(q_ξ)
     m_out, v_out   = mean_cov(q_out)
-    return (ndims(q_out) * log2π - chollogdet(mean(q_Λ)) + tr(mean(q_Λ) * (v_out + v_mean + (m_out - m_mean) * (m_out - m_mean)'))) / 2
+    return @views (ndims(q_out) * log2π - mean(logdet, q_Λ) + tr(mean(q_Λ) * (v_out + v_mean + (m_out - m_mean) * (m_out - m_mean)'))) / 2
```

Comment:
I have some concerns as to whether this rule is actually correct (likely my bad). Just did the derivations again and seem to end up with
ndims(q_out) * log2π - mean(chollogdet, q_Λ) + tr(mean(q_Λ) * (v_out + v_mean + m_mean*m_mean'+ m_out * m_out')) - 2*m_out*m_mean ) / 2
I might have a mistake somewhere, but we better double check.

Reply:
I've changed it to a different form.